feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/AptoideAppsRepository.kt

```diff
@@ -183,6 +184,13 @@ internal class AptoideAppsRepository @Inject constructor(
         ?: throw IllegalStateException()
     }
 
+  override suspend fun getAppsDynamicSplits(md5: String): List<DynamicSplit> =
+    withContext(scope.coroutineContext) {
```

Comment:
why running this code scoped if its already a suspend function?

Reply:
This should be reviewed in a separate ticket that evaluates the scope in all repositories